### ISSUE TITLE
Issue with Sphinx 4

### ISSUE THREAD

**Issue by lelit** [sphinx-contrib/plantuml#56]
Trying out Sphinx 4.0.0b2, I got the following showstopper:
```
Running Sphinx v4.0.0b2
loading translations [it]... done

Extension error:
Could not import extension sphinxcontrib.plantuml (exception: cannot import name 'ENOENT' from 'sphinx.util.osutil' (/usr/local/lib/python3.8/site-packages/sphinx/util/osutil.py))
```

See this [commit](https://github.com/sphinx-doc/sphinx/commit/a86346aca6bf99a8920da366caaad7c47809ecce#diff-e320b3cc1562e9e03d8208cc8871cec31f5bc7e37529d9cc0ec22743b167c2f3).

**Comment by t-animal** [comment]
Would it be possible to make a release including this fix? As sphinx 4.0 was released yesterday,  I suspect many pipelines (including ours) are failing.

In the meantime: it helps explicitly installing `pip install sphinx==3.5.4`

**Comment by yuja** [comment]
Tagged 0.21, thanks for the reminder.
